rest-service/manager_rest/test/endpoints/test_deployments.py

```diff
@@ -944,6 +944,43 @@ def test_creation_failure_with_invalid_labels(self):
                                deployment_id=resource_id,
                                labels=self.INVALID_LABELS)
 
+    @attr(client_min_version=3.1,
+          client_max_version=base_test.LATEST_API_VERSION)
+    def test_creation_failure_with_invalid_label_key(self):
+        resource_id = 'i{0}'.format(uuid.uuid4())
+        err_label = [{'k ey': 'value'}]
+        error_msg = '400: .*The label\'s key {0} contains illegal ' \
+                    'characters'.format('k ey')
```

Comment:
This is way too exact
```suggestion
        error_msg = 'k ey.*illegal characters'
```
just this would be enough, ditto the other test

Reply:
Right.